src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -3,32 +3,30 @@ import { connect } from 'react-redux';
 import Modal from '@department-of-veterans-affairs/formation-react/Modal';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import { selectCurrentlyOpenEditModal } from '../selectors';
-import { openModal, createTransaction } from '../actions';
+import {
+  openModal,
+  createTransaction,
+  updateSelectedAddress,
+} from '../actions';
 
 import * as VET360 from '../constants';
 
 class AddressValidationModal extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      selectedOption: '',
-      address: {},
-    };
-  }
-
-  onChangeHandler = address => event => {
-    this.setState({ selectedOption: event.target.name, address });
+  onChangeHandler = (address, selectedId) => _event => {
+    this.props.updateSelectedAddress(address, selectedId);
```

Comment:
This prop is probably required, but if `updateSelectedAddress` is not passed this could cause a runtime error 👍

Reply:
@kelsonic its mapped in mapDispatchToProps